style.css

```diff
@@ -1,6 +1,6 @@
 @import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
```

Comment:
[OPTIONAL] I think your font-weight is not working properly. Make sure that when you import from google fonts you actually select every font weight you are going to use. 

This is an example of some font with weights that I have:

![Captura de Pantalla 2022-09-28 a la(s) 7 52 13 a  m](https://user-images.githubusercontent.com/86022690/192783354-62728df6-be3d-4bf4-97ba-0311947f87d0.png)

Hope this helps 🤓